app/scripts/administration/directives/location-authorization.directive.js

```diff
@@ -142,6 +156,19 @@
     }
 
 
+    function selectAllAggregateLocations() {
+      if ($scope.selectedLocationsAggregateData.selectedAllAgg === false) {
+        $scope.selectedLocationsAggregateData.selectedAllAgg = true;
+        $scope.selectedLocationsAggregateData.locations = $scope.locations;
+      }
+      else {
+        $scope.selectedLocationsAggregateData.selectedAllAgg = false;
```

Comment:
Well it should be properly named then. It is confusing to name a function with this very clear descriptive name only to find out it does something different.